packages/tools/test/apm/uninstall.coffee

```diff
@@ -0,0 +1,19 @@
+
+nikita = require '@nikitajs/core'
+{tags, ssh} = require '../test'
+they = require('ssh2-they').configure ssh...
+
+return unless tags.tools_apm
+
+describe 'tools.apm.uninstall', ->
+
+  they 'uninstall apm package using tools.apm.uninstall', ({ssh}) ->
+    nikita
+      ssh: ssh
+    .tools.apm.install
+      name: 'package-list'
+    .tools.apm.uninstall
+      name: 'package-list'
+    , (err, {status}) ->
+      status.should.be.true() unless err
```

Comment:
You are not testing to run `uninstall` on a package which is not installed (status `false`)

Reply:
Can you check this again David? The first `apm.install` is preparation for the `apm.uninstall` test. The test status is `true` because we can be sure that there was an installation to uninstall.

Reply:
As I wrote, you are testing the uninstallation of an installed package, you are not testing the uninstallation of an uninstalled package.

Reply:
The first action installs it though.

Reply:
1. we are not testing the installation
2. don't tell me the package is installed when I wrote you that we want to write a test against a package which is not installed

Reply:
Ok so we change the test to run `apm.uninstall` on a package which is not installed (I wrote the test wanting to `uninstall` a package which was definitely `installed`)

To test the uninstallation of an not installed package, would it not be clearer to have two steps, uninstall then uninstall again? Otherwise, it not clear that the initial state is that package-list isn't installed.

Reply:
I didn't asked you to change this test, I just notice that you are not testing an important scenario. Thus you must write a second test. Yes, you can call uninstall twice and only validate the second call